content/smart.md

```diff
@@ -0,0 +1,140 @@
+
+- Providers can access SMART apps from the *Cerner Millennium PowerChart* TOC (table of contents, a column of options that open different PowerChart components) and the Organizer view
+- Providers can access SMART apps from an *MPages* component that is embedded in *PowerChart*
+- Consumers can access SMART apps from HealtheLife to view their patient information
+Cerner is continuing to look at incorporating SMART app support into our other applications and platforms.
+
+### SMART App Data Retrieval ###
+
+SMART apps can leverage Cerner's implementation of the FHIR standard server to retrieve data from the EHR based upon the current patient in context. However, SMART apps are not limited to FHIR data for their needs. SMART apps may choose to leverage other data sources for their needs, from other third party data sources to private and proprietary data the SMART app maintains. SMART apps may even retrieve data through Cerner proprietary means (for example, MillenniumObjects, CareAware services, or CCL scripts). Of course, leveraging proprietary APIs hampers interoperability, but if that is not a concern of the SMART app, it is a viable option.
+
+### SMART Without FHIR ###
+
+Technically speaking, the SMART specification requires that a FHIR server exist so that a SMART app can communicate with the FHIR server when the app opens in order to determine the location of the authorization (OAuth 2) server. However, outside of this requirement, FHIR does not need to be leveraged otherwise by the SMART app.
+
+## Technical ##
+
+### Screen Resolution ###
+
+The recommended screen resolution varies based on the available screen's real estate. Instead of targeting a specific resolution, the app should be designed to be responsive, and the app should adjust according to the available screen sizes.
+
+### Browser Requirement ###
+
+Apps should be designed to work in any browser for best user experience. For provider facing apps that run on the *Cerner Millennium* platform, Cerner currently supports only the embedded Internet Explorer browser (IE) at the moment. The minimum-supported embedded browser is IE 10. The latest supported browser varies based on the version of the browser that is currently installed on the device.
+
+### HTML5 DOCTYPE ###
+
+The doctype declaration tells the browser what version of HTML the document is written in. It is a good idea to include this declaration so that the browser will render the document correctly. It must be included at the very first line of the document, right above the `<html>` tag. 
+
+```<!DOCTYPE html>```
+
+### X-UA-Compatible Tag ###
+
+In order for the embedded IE browser to render the app in the latest supported IE version, set the following HTML tag in the first line of the `<head>` section for all pages:
+
+```<meta http-equiv="X-UA-Compatible" content="IE=edge">```
+
+This is required because the default embedded IE version is set to another version for compatibility with other solutions.
```

Comment:
I would replace this line with:

"If this tag is not present as the first `<head>` element on a page, the page and all associated JavaScript will be rendered in IE 7 mode, often times resulting in errors and unwanted behavior."